docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,24 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129)</sup>
+1. Create or import cluster into Rancher
+2. Go into the cluster
+3. Click Longhorn from the left sidebar
+4. Open Longhorn UI from the Rancher UI
```

Comment:
Can we rephrase this as `Click the Longhorn app in Rancher to navigate to the Longhorn UI`?

Reply:
I was thinking of the case when install through kubectl, I don't know if this will still be called as an "app", but sure, I can rephrase this section.